modules/slack/blocks/block_kit_examples/mentorship/mentorship_request_block.json

```diff
@@ -0,0 +1,588 @@
+
+{
+	"blocks": [
+		{
+			"type": "section",
+			"block_id": "mentorship_request_title",
+			"text": {
+				"type": "mrkdwn",
+				"text": ":male-teacher: Mentor Request Form :female-teacher:\n Thank you for signing up for a mentoring session. Please fill out the form below. If you have any questions, please don't hesitate to contact @mentor-coordinators.",
+				"verbatim": false
+			}
+		},
+		{
+			"type": "divider",
+			"block_id": "mentorship_divider_1"
+		},
+		{
+			"type": "section",
+			"block_id": "mentor_service",
+			"text": {
+				"type": "mrkdwn",
+				"text": "*Type of Mentorship Service*",
+				"verbatim": false
+			},
+			"accessory": {
+				"type": "static_select",
+				"action_id": "mentor_service_select",
+				"placeholder": {
+					"type": "plain_text",
+					"text": "Type of Service",
+					"emoji": true
+				},
```

Comment:
If I recall correctly, the old bot didn't actually match this selection with anything in Airtable. Does this new rewrite actually make this part useful?

If not, is the plan to do that in a follow up? If not, I suggest that we migrate the missing ones to "Skillsets" and then get rid of this as an extra field. Either way is fine tho

Reply:
Yeah, I added in a type of service table in Airtable. The list of tables is in the Wiki - https://github.com/OperationCode/operationcode-pybot/wiki/Airtable-Usage-and-Notes